src/backend/distributed/operations/shard_cleaner.c

```diff
@@ -195,10 +152,26 @@ citus_cleanup_orphaned_resources(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * isolation_cleanup_orphaned_resources implements a test UDF that's the same as
+ * citus_cleanup_orphaned_resources. The only difference is that this command can
+ * be run in transactions, this is to test
```

Comment:
missing part of the comment (it already had this problem in the original comment that you copy pasted).
```suggestion
 * be run in transactions, this is needed to test this function in isolation tests since commands are automatically run in transactions there.
```